slackbot/dispatcher.py

```diff
@@ -138,9 +138,14 @@ def loop(self):
         while True:
             events = self._client.rtm_read()
             for event in events:
-                if event.get('type') != 'message':
-                    continue
-                self._on_new_message(event)
+                if event.get('type') == 'message':
```

Comment:
That is something I can do soon for you :) Not sure if it makes it perform any better in terms of CPU/Memory though. (Probably better for CPU, worse for Memory at a rough guess?)